include/avr/sfr_defs.h

```diff
@@ -251,7 +251,7 @@
 
     Wait until bit \c bit in IO register \c sfr is set. */
 
-#define loop_until_bit_is_set(sfr, bit) do { } while (bit_is_clear(sfr, bit))
+#define loop_until_bit_is_set(sfr, bit) do { } while (bit_is_set(sfr, bit))
 
```

Comment:
This is wrong. When you want to loop until a bit is set (waiting for a bit to be set), then the loop must not be left when the bit is clear.  Similar applies to the hunk below.